index.html

```diff
@@ -3,32 +3,26 @@
 ---
 <!--
 ==================================================
-Slider Section Start
+Typewriter Section Start
 ================================================== -->
 <section id="hero-area">
-    <div class="container">
-        <div class="row">
-            <div class="col-md-12 text-center">
-                <div class="block wow fadeInUp" data-wow-delay=".3s">
-                    <!-- slider -->
-                    <section class="cd-intro">
-                        <h1 class="wow fadeInUp animated cd-headline slide" data-wow-delay=".4s">
-                        <span>We are Clemson Cyber</span><br>
-                        <span class="cd-words-wrapper">
-                            <b class="is-visible"><span style="color: blue;">Defense</span></b>
-                            <b><span style="color: red;">Offense</span></b>
-                            <b><span style="color: purple;">Security</span></b>
-                        </span>
-                        </h1>
-                    </section>
-                    <h2 class="wow fadeInUp animated" data-wow-delay=".6s">
-                        Training the next generation of cyber security experts
-                    </h2>
-                    <a href="/about" class="btn btn-default btn-contact wow fadeInUp" data-wow-delay=".9s">About Us</a>
-                    <!-- /slider -->
-                </div>
-            </div>
+  <div class="container">
+    <div class="row">
+      <div class="col-md-12 text-center">
+        <div class="block wow fadeInUp" data-wow-delay=".3s">
+          <section class="cd-intro">
```

Comment:
I'm pretty sure `cd-intro` is no longer necessary as nothing defines it now (well actually it wasn't in slider.css before but all of the `cd-` ones seemed to be there).

```suggestion
          <section>
```

Reply:
might as well get rid of that `<section>` tag altogether since there's no CSS bound to it then.